src/gt4py/lazy_stencil.py

```diff
@@ -77,3 +78,18 @@ def __call__(self, *args: Any, **kwargs: Any) -> None:
     def run(self, *args: Any, **kwargs: Any) -> None:
         """Pass through to the implementation.run."""
         self.implementation.run(*args, **kwargs)
+
+    def __sdfg__(self, **kwargs):
```

Comment:
Does this need a `__deepcopy__`?

Reply:
You mean the class needs a custom deep copy functionality? Why so?

Reply:
Ah, because `StencilObject` has one.

Reply:
Nevermind, I understand now. All good!